docs/content/providers/docker.md

```diff
@@ -116,6 +116,20 @@ Ports detection works as follows:
   by using the label `traefik.http.services.<service_name>.loadbalancer.server.port`
   (Read more on this label in the dedicated section in [routing](../routing/providers/docker.md#port)).
 
+### Host networking
+
+When exposing containers that are configured with [host networking](https://docs.docker.com/network/host/),
+the IP address of the host is resolved as follows:
+
+<!-- - if in swarm mode, check whether the Node.IPAddress field of the container is provided by the API -->
```

Comment:
this review comment is just for blocking merging until we all agree on what to do about the Node.IPAdress case.